recipes/eukulele/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.2" %}
+{% set sha256 = "c21a85a8ea2e4a88ff9fc6828ab08e1ed6f05fbf45c94951948a219f0467891a" %}
+
+package:
+  name: eukulele
+  version: 1.0.2
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+source:
+  url: https://github.com/AlexanderLabWHOI/EUKulele/archive/v1.0.2z.tar.gz
+  sha256: {{ sha256 }}
+
+test:
+  commands:
+    - EUKulele --version | grep '{{ version }}'
+    - EUKulele --help
+
+requirements:
+  host:
+    - python>=3.6
+    - pip
+    - pytest-runner
+  run:
+    - python>=3.6
+    - numpy
+    - biopython
+    - pandas
+    - seaborn
+    - chardet
+    - joblib
+    - ujson
+    - pyyaml
+    - matplotlib-base
+    - blast 
```

Comment:
Maybe specifying minimum blast and diamond versions will help in solving the environment.

Reply:
For some reason this doesn't seem to be helping with the time to build - conflicts prevented me from being highly specific with the minimum version, but specifying some minimum doesn't seem to have helped with the time taken. Thanks so much for your help with this!